js/build.js

```diff
@@ -101,7 +103,7 @@ Fliplet.Widget.instance('image-gallery', function (data) {
 
   // Appearance change Hook
   Fliplet.Hooks.on('appearanceChanged', function () {
-    initGallery();
+    initGallery('appearance');
```

Comment:
Instead of passing `appearance` as the context of "how" the function was called, pass a parameter to the `initGallery()` function that says how it should behave differently.

**Action** functions should generally follow this rule, e.g. `initGallery({ append: true })`. I'm not sure what the difference is here so I can't recommend what the parameter name should be.

Reply:
@tonytlwu 
Fixed